lib/plugins/healthcheck/commonHealthcheckPlugin.ts

```diff
@@ -0,0 +1,179 @@
+import type { Either } from '@lokalise/node-core'
+import type { FastifyPluginCallback } from 'fastify'
+import fp from 'fastify-plugin'
+import type { AnyFastifyInstance } from '../pluginsCommon'
+import type { HealthChecker } from './healthcheckCommons'
+
+export interface CommonHealthcheckPluginOptions {
+  responsePayload?: Record<string, unknown>
+  logLevel?: 'fatal' | 'error' | 'warn' | 'info' | 'debug' | 'trace' | 'silent'
+  healthChecks: readonly HealthCheck[]
+  infoProviders?: readonly InfoProvider[]
```

Comment:
do we have a use-case for these values?

Reply:
I haven't seen usages, but I'm also not familiar with all services we have

Reply:
I guess there is no harm in having it, it's optional anyway